feedback-module/src/services/interactionService.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
```

Comment:
Where is the interactionService being called?

Reply:
forgot to delete it—i'll update the PR